LitvaKebabs/Components/PostcodeLookup.razor.cs

```diff
@@ -0,0 +1,103 @@
+﻿using LitvaKebabs.Models;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using Newtonsoft.Json;
+using RestSharp;
+using System.Web;
+
+namespace LitvaKebabs.Components
+{
+    public partial class PostcodeLookup
+    {
+        private const string deliveringFromPostcode = "BH243FW";
+        private const double deliveringFromPostcodeLon = -1.775614;
+        private const double deliveringFromPostcodeLat = 50.84143;
```

Comment:
keep in mind, i have no idea what im looking at here, so for all i know you could be doing this already. but could you impliment something like the Haversine formula to calculate the shortest possible distance between the verified customer postcode and the deliveringFromPostcode?

Reply:
I think this is what GetDistance() is doing...I don't know lol